en/content_how_to_get_compensated_for_work.md

```diff
@@ -1,47 +1,59 @@
+## Invoicing and Payment System
 
-* funding a project
-* conference travel
-* charitable donations to an endowment in lieu
-* credits for compute resources, archive nodes, etc
-* supporting a blockchain-related organization or event of your choice
+SCRF officially pays invoices once per month at the end of the month. SCRF may process another round of payments mid-month, in certain situations. **All invoices are due two full business days before the end of the month**. Any billable hours forgotten or worked after the invoice was submitted can be added to the next invoice with a description. SCRF uses Request Network for all payments.
 
-Pay varies according to the complexity of the assignment and the time involved. Contributors who commit to longer-term engagements and have produced a good body of work for the forum are encouraged to seek additional and more lucrative responsibilities.
+* Use an Ethereum address. (SCRF recommends [Metamask](https://metamask.io/) but any exchange wallet is acceptable when transferring funds to a fiat wallet right away.)
+* Create an account on [Request Network](https://app.request.finance/login).
+* Choose USD for invoice currency.
+* Select DAI stablecoin as the payment cryptocurrency.
+* Add a new client using the email address invoice@scrf.io.
+  * Smart Contract Research Forum
+  * 1250 Broadway
+  * 36th Fl
+  * 10001 New York
+  * _Note: Other fields can be left blank._
+* Provide full name and chat username in the memo field.
+* Attach a copy of your version of [Your Name] Hours Template Sheet or provide a link to the sheet.
+  * For those submitting bounties, it is not required to include hours but please include a link to your submission.
+* Submit the invoice for payment.
 
-## Tracking Hours
+## Compensation & Logistics
 
-SCRF pays contributors by the hour. Hours are tracked in [a spreadsheet](https://docs.google.com/spreadsheets/d/19CmaqELKS_S2gWQIs2kddA91rv5zca-51mk_sOJ0gYs/edit?usp=sharing) that is submitted once a month with your invoice (see below). Please share a copy of this sheet with your team lead.
+If unable to accept funds directly, alternative compensation options could include:
 
-## Invoicing and Payment System
+* Funding a project
+* Conference travel
+* Charitable donations to an [endowment](https://thegivingblock.com/donate/)
+* Credits for compute resources, archive nodes, etc.
+* Direct support of a blockchain-related organization or event of choice.
+
+## Notes
+
+For questions, concerns, or to make alternative arrangements, contact SCRF’s Operations Team in a [Discord chat](https://discord.com/channels/784234332617048065/784234333111451670).
+
+* Please note that participation in community calls, events, and cohorts are not billable.
+* Invoices are due two full business days before the end of each month. Make sure to keep an eye on the Discord chat for any announcements, such as holidays, that may impact the invoice due date.
```

Comment:
This is out of date. It should be 5 days. Please coordinate with @vincenzospaghetti about the current billing cycle.

Reply:
Hello, this is an example.